lib/src/main/java/vmath/Mat3.java

```diff
@@ -44,4 +44,16 @@ public float get(int i, int j) {
         }
         return m[i * 3 + j];
     }
+
+    public void transp() {
+        float[] result = new float[9];
+
+        for (int i = 0; i < 3; i++) {
+            for (int j = 0; j < 3; j++) {
+                result[i * 3 + j] = m[j * 3 + i];
```

Comment:
Please follow the code style that we follow. The rule is to not use spaces around multiplication operators:
`Use spaces around arithmetic binary operators, except for multiplicative operations (*, /, %).`